src/screens/IndexScreen.js

```diff
@@ -365,67 +375,88 @@ export const IndexScreen = ({ navigation, route }) => {
                 : undefined;
 
             return (
-              <ListComponent
-                ListHeaderComponent={
-                  <>
-                    {!!showFilter && (
-                      <>
-                        <DropdownHeader
-                          {...{
-                            query,
-                            queryVariables,
-                            data: initialNewsItemsFetch && loading ? {} : data,
-                            updateListData: updateListDataByDropdown
-                          }}
+              <>
+                {isCalendar && !hasDailyFilterSelection && (
+                  <CalendarListToggle
```

Comment:
Why is there a hasDailyFilterSelection control? Is it desirable that the toggle disappears when this filter is active?
|list toggle - filter: false|list toggle - filter: true|calendar toggle - filter: false|calendar toggle - filter: true|
|--|--|--|--|
![IMG_2040](https://user-images.githubusercontent.com/11755668/214007742-d6038a4f-0bbf-4ee9-8f73-3ee71740a8b6.PNG) | ![IMG_2041](https://user-images.githubusercontent.com/11755668/214007785-a2844654-36b9-4973-91b6-8a0691147966.PNG) | ![IMG_2042](https://user-images.githubusercontent.com/11755668/214007806-ab3f8b5f-caea-493e-915e-9b1777f0a0ff.PNG) | ![IMG_2043](https://user-images.githubusercontent.com/11755668/214007824-8b43d12c-d0e9-41e2-98ba-82719952aec9.PNG)

Reply:
good point. with daily filter we should show the list always, right? even if we have selected the calendar before.
or we keep the toggle visible as well and let the users handle and know everything on their own?

Reply:
Yes, but neither is possible at the moment. I have activated the daily filter after opening the calendar view. 

Calendar appears. toggle does not appear.

According to the ticket, if the daily event filter is activated, the list view should be switched to. So I think it would be better to remove the toggle and replace the calendar with the list.

Or am I thinking wrong?

Reply:
i think we agree on this:

https://user-images.githubusercontent.com/1942953/214077838-a7c3ec89-acf1-4336-b4fd-ff92a63ba684.mp4

Reply:
Yeah, it's better now.  👍